packages/rnv-deploy-aws/index.js

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable global-require, import/no-dynamic-require */
+import path from 'path';
+import { CHROMECAST, WEB } from '../rnv/src/constants';
```

Comment:
I would use the same import method as the one for config for example, for anything that's related to RNV. This path will not work for users